code/__DEFINES/misc.dm

```diff
@@ -155,11 +155,34 @@
 #define ASSEMBLY_UNLOCKED 1
 #define ASSEMBLY_LOCKED 2
 
+// RESEARCH UPGRADES DEFINES //
+
 // Matrix CAS Upgrades
 #define MATRIX_DEFAULT 0
 #define MATRIX_NVG 1
 #define MATRIX_WIDE 2
 
+#define UPGRADE_EXCLUDE_BUY 0
+#define UPGRADE_CATEGORY 0.5 //lord forgive me
+#define UPGRADE_TIER_1 1
+#define UPGRADE_TIER_2 2
+#define UPGRADE_TIER_3 3
+#define UPGRADE_TIER_4 4
+#define UPGRADE_TIER_5 5
```

Comment:
add some prefix to these to indicate what they're meant for (if they were for the autodoc, add `AUTODOC_` or similar)

Reply:
I left them in "general" names because eventually there will be more items that have same behavior. I didnt specify it was for research upgrades, fixed that.